src/main/java/com/gui/pages/desktop/ProductsListPage.java

```diff
@@ -11,32 +12,36 @@
 public class ProductsListPage extends AbstractPage {
     protected WebDriver driver;
 
-    @FindBy (xpath = "//div[contains(@class, 'thumbnails grid row list-inline')]//a[contains(@class, 'prdocutname')]")
-    private List<WebElement> itemContainerList;
+    @FindBy(xpath = "//div[contains(@class, 'thumbnails grid row list-inline')]//a[contains(@class, 'prdocutname')]")
+    private List<SearchItem> itemContainerList;
```

Comment:
This locator is used to find the product name of the component, not the component itself.